libs/entitlement/build.gradle

```diff
@@ -9,12 +9,25 @@
 apply plugin: 'elasticsearch.build'
 apply plugin: 'elasticsearch.publish'
 
+apply plugin: 'elasticsearch.embedded-providers'
+
+embeddedProviders {
+  impl 'entitlement', project(':libs:entitlement:asm-provider')
+}
+
+configurations {
+  entitlementBridge
```

Comment:
I don't see any references to this. I'm not sure why it's here; perhaps left over from prior iterations of this PR.